src/main/python/SentimentAnalyzer.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+# pip install --upgrade azure-ai-textanalytics
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+
+_key = "adc96f39ac2a4918a1f462b5bf8674b8"
+_endpoint = "https://nbatwittersentiment.cognitiveservices.azure.com/"
+
+def _authenticate(key:str,endpoint:str):
+    # Authentication
+    ta_credential = AzureKeyCredential(key)
+    client = TextAnalyticsClient(
+        endpoint=endpoint,
+        credential=ta_credential,
+        api_version='TextAnalyticsApiVersion.V3_0')
+    return client
+
+def _formatresponse(responseDict) -> pd.DataFrame :
```

Comment:
Can you add some docs for at least what the responseDict is? I think that's the dict of tweetID, tweet, but I'm not sure. Also, given our conversation earlier today, that dict may become date, tweet instead since the date would be a unique identifier.
Overall it's a bit hard to understand this method. It looks like responseDict may not be a dictionary but some other sort of object?

Reply:
Jk, date can't be used as a key because of mixing different players together into the same df/dictionary. The key will stay tweetID.

Reply:
Added doc from Microsoft, it is a list of objects.